test/algorithms/perimeter/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+# Boost.Geometry
+# Copyright (c) 2024 Barend Gehrels, Amsterdam, the Netherlands.
+# Use, modification and distribution is subject to the Boost Software License,
+# Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
+# http://www.boost.org/LICENSE_1_0.txt)
+
+foreach(item IN ITEMS
+    perimeter
+    perimeter_base
```

Comment:
These two above (were duplicate) are quite similar, and quite short, we can merge them later.
But I wanted this PR to be CMake only.